crates/nu-command/tests/commands/let_.rs

```diff
@@ -46,6 +46,13 @@ fn mut_pipeline_allows_in() {
     assert_eq!(actual.out, "21");
 }
 
+#[test]
+fn let_pipeline_redirects() {
+    let actual = nu!(r#"let x = echo 'bar'; $x | str length"#);
```

Comment:
Maybe worth having the test with a testbin as an external as well. Just to make sure we don't break if we decide to change the `echo` behavior.

Reply:
Added.